src/test/java/fi/otavanopisto/kuntaapi/test/LinkedEventsEventMocker.java

```diff
@@ -0,0 +1,108 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.tomakehurst.wiremock.matching.StringValuePattern;
+
+import fi.metatavu.linkedevents.client.model.Event;
+
+@SuppressWarnings ("squid:S1166")
+public class LinkedEventsEventMocker extends AbstractBaseMocker {
+
+  private static final String PATH_TEMPLATE = "%s%s/";
+  private static final String RESOURCES_PATH = "/v1/event/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)